facia/app/services/dotcomrendering/FaciaPicker.scala

```diff
@@ -112,6 +126,7 @@ object FaciaPicker extends GuLogging {
       ("hasNoSlideshows", FrontChecks.hasNoSlideshows(faciaPage)),
       ("hasNoPaidCards", FrontChecks.hasNoPaidCards(faciaPage)),
       ("hasNoRegionalAusTargetedContainers", FrontChecks.hasNoRegionalAusTargetedContainers(faciaPage)),
+      ("hasNoSnapLinkCards", FrontChecks.hasNoSnapLinkCards(faciaPage)),
```

Comment:
Great finding! After this gets merged, shall we add a visualisation for this check in the dashboard? https://metrics.gutools.co.uk/d/pCbViPIVk/front-migration?orgId=1